content/blog/2019/08/2019-08-13-remoting-kafka-kubernetes-release-2.0.adoc

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Remoting over Apache Kafka plugin release 2.0"
+tags:
+- remoting
+- kafka
+- kubernetes
+- helm
+- developer
+- gsoc
+- gsoc2019
+author: longnguyen
+---
+
+= Remoting over Apache Kafka plugin release 2.0
+
+I am Long Nguyen from FPT University, Vietnam. My project for Google Summer of Code 2019 is link:https://jenkins.io/projects/gsoc/2019/remoting-over-apache-kafka-docker-k8s-features/[Remoting over Apache Kafka with Kubernetes features]. After a successful link:https://jenkins.io/blog/2019/07/11/remoting-kafka-kubernetes-phase-1/[Phase 1], finally the 2.0 version of the plugin has been released in Phase 2. The 2.0 version provides seamless integration with Kubernetes environment.
+
+== 2.0 version features
+
+* Start a simple Kafka server in Kubernetes.
+
+* Dynamically provision Remoting Kafka Agent in Kubernetes.
+
+* Helm chart to bootstrap the whole system in Kubernetes.
+
+== Start a simple Kafka server in Kubernetes
+
+Using the plugin requires users to have a configured Zookeeper and Kafka server, which would be intimidating for people who just want to try out the plugin. Now, users can start a simple, single-node Kafka server in Kubernetes environment with just one button click.
+
+image:/images/post-images/remoting-kafka/kafka-provisioning-kubernetes-ui.png[Kafka provisioning in Kubernetes UI]
+
+On the Global Configuration page, users can input Kubernetes server information and credentials. When users click `Start Kafka on Kubernetes` button, Jenkins will create a Kubernetes client from the information and then apply Zookeeper and Kafka YAML specification files from `resources`. After downloading images and creating containers, it will automatically update Zookeeper and Kafka URLs into respective fields.
+
+== Dynamically provision Remoting Kafka Agent in Kubernetes
+
+With previous version, users have to manually add/remove nodes so it is hard to scale builds quickly. link:https://github.com/jenkinsci/kubernetes-plugin[Kubernetes plugin] allows us to dynamically provision agents in Kubernetes but it is designed for JLPT agent. With this new version, Remoting Kafka agent can also be provisioned automatically in Kubernetes environment.
+
+image:/images/post-images/remoting-kafka/cloud-ui.png[Remoting Kafka Cloud UI]
+
+Users can find the new feature in Cloud section. Here users could input Kubernetes connection parameters and desired Remoting Kafka agent properties including labels. When new build with matching labels get started and there is not any free nodes, Cloud will automatically provision Remoting Kafka agent pod in Kubernetes to run the build.
```

Comment:
Is it possible to link to localhost of Cloud section?

Reply:
"get started" -> "gets started"
"and there is not any free" -> "and there are no free"